sycl/unittests/assert/assert.cpp

```diff
@@ -357,12 +359,14 @@ static pi_result redefinedProgramGetInfo(pi_program P,
   if (PI_PROGRAM_INFO_DEVICES == ParamName) {
     EXPECT_EQ(ParamValueSize, 1 * sizeof(cl_device_id));
 
-    cl_device_id Dev = Device->get_native<sycl::backend::opencl>();
+    cl_device_id Dev = sycl::get_native<sycl::backend::opencl>(*Device);
 
-    if (ParamValue)
+    if (ParamValue) {
```

Comment:
According to the [LLVM Coding Standard](https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements) braceless is preferred for single-expression control-flow statements, unless it makes it harder to read. Personally I don't mind and I don't think we adhere to it many places, but it is something to think about.

Reply:
@steffenlarsen Thank you for the comment, I won't insert `{ }` in one-line control-flow statements anymore.

Reply:
@psamolysov-intel, please, revert this change.

Reply:
Reverted.